### DISCUSSION TITLE
Allow opening an environment json file and sync with that aside from Import/Export

### DISCUSSION THREAD

**Discussion by louiealmeda** [mockoon/mockoon#467] [Ideas] [answered]
I think it will be extremely helpful if Mockoon App can write directly to an `opened` file, instead of exporting it every time we have to save it.
  
We have our mockoon environment as json in our Git repository. So we get the benefit of versioning our mock apis.
Currently, the workflow looks like this:

1. Git pull
2. Open Mockoon and import the mockoon environment.json 
3. Make changes to the mocks using the mockoon App
4. Export environment.json, overwrite the existing environment.json we have in source.
5. Commit that version / run the mocks with cli.

ideally, the work flow would be:
1. Git pull
2. Open the environment.json as file (not import) if not yet opened.
3. Mockoon automatically overwrites the file as we change the environment using the App
4. Commit the version / run the mocks with cli
  
I'm happy to work on this if this is something we are looking to do.
Thanks.

**Comment by 255kb** [comment, marked as answer]
Thanks for the suggestion @louiealmeda 
The storage system is quite basic now, and didn't change since I created the app 3 years ago 😅 
We are currently working on this subject, and changes are coming this year. We are probably heading towards what you describe. 
This change is tracked by this issue: https://github.com/mockoon/mockoon/issues/16

Also, thank you for proposing your help but we won't be accepting any external contribution for this feature/change. It involves too many changes and can have many impacts on the future of the application. 😃

**Reply by louiealmeda** [reply]
Awesome, looking forward to it! 🙂
